packages/ndla-image-search/src/ImageSearchResult.tsx

```diff
@@ -37,33 +53,40 @@ export default function ImageSearchResult({
   showCheckbox,
   checkboxLabel,
 }: Props) {
-  const active = selectedImage && selectedImage.id === image.id ? "active" : "";
+  const active = selectedImage?.id === image.id;
+
+  const onKeyPress = (e: KeyboardEvent<HTMLButtonElement>) => {
+    if (e.key === "Enter") {
+      onImageClick(image);
+    }
+  };
 
   return (
-    <div key={image.id} className={`list-item ${active}`}>
-      <div className="list-item-inner">
-        <StyledButton variant="stripped" data-testid="select-image-from-list" onClick={() => onImageClick(image)}>
-          <img
-            role="presentation"
-            alt="presentation"
-            srcSet={getPreviewSrcSets(image.image.imageUrl)}
-            src={image.image.imageUrl}
-          />
-          <span className="list-item-title">{image.title.title}</span>
-        </StyledButton>
-      </div>
-      {selectedImage && selectedImage.id === image.id ? (
+    <>
+      <StyledButton
+        variant={active ? "secondary" : "tertiary"}
+        data-testid="select-image-from-list"
+        onClick={() => onImageClick(image)}
+        onKeyDown={onKeyPress}
+        aria-expanded={active}
+        aria-controls={`image-preview-${image.id}`}
```

Comment:
Burde denne ha en aria-label?

Reply:
Jeg var litt i tvil, men tenker kanskje nei, tror du ikke det blir slitsomt med en tekst som leses opp for x antall søkeresultater?